roles/download/tasks/download_container.yml

```diff
@@ -39,3 +40,18 @@
     - download.container
     - pull_required|default(download_always_pull)
     - group_names | intersect(download.groups) | length
+    - container_manager != "containerd"
+
+- name: container_download | Download containers if pull is required or told to always pull (all nodes)
```

Comment:
Can we add `containerd` in this task name?

Reply:
Sure, see last push. I'll be happy to add a CI job if you help out, I'll contact you on slack.